src/views/PendingReviewDetail.vue

```diff
@@ -0,0 +1,279 @@
+            </ion-label>
+            <ion-button slot="end" fill="outline" color="medium">{{ translate("Rename") }}</ion-button>
+          </ion-item>
+          <ion-chip outline>
+            <ion-icon :icon="calendarClearOutline"></ion-icon>
+            <ion-label>3rd March 2024</ion-label>
+          </ion-chip>
+          <ion-chip outline>
+            <ion-icon :icon="businessOutline"></ion-icon>
+            <ion-label>Broadway</ion-label>
+          </ion-chip>
+        </div>
+        <ion-list>
+          <div class="filters ion-padding">
+            <ion-item>
+              <ion-spinner slot="start" name="circular" paused="true"/>
+              <ion-label>{{ translate("Progress") }}</ion-label>
+              <ion-label slot="end">40% complete</ion-label>
+            </ion-item>  
+            <ion-item>
+              <ion-spinner slot="start" name="circular" paused="true"/>
+              <ion-label>{{ translate("Variance") }}</ion-label>
+              <ion-label slot="end">13 counted | 20 expected</ion-label>
+            </ion-item> 
+            <ion-item lines="none">
+              <ion-icon slot="start" :icon="thermometerOutline"/>
+              <ion-label>{{ translate("Item variance threshold") }}</ion-label>
+              <ion-label slot="end">50%</ion-label>
+            </ion-item>  
+            <ion-item lines="none">
+              <ion-range aria-label="Range"></ion-range>
+            </ion-item>
+          </div>
+        </ion-list>
+      </div>
+
+      <div class="header">
+          <ion-segment>
+            <ion-segment-button>
+              <ion-label>{{ translate("All") }}</ion-label>
+            </ion-segment-button>
+            <ion-segment-button>
+              <ion-icon color="success" :icon="thermometerOutline"/>
+            </ion-segment-button>
+            <ion-segment-button>
+              <ion-icon color="danger" :icon="thermometerOutline"/>
+            </ion-segment-button>
+          </ion-segment>
+      </div>
+
+      <hr/>
+
+      <main>
+        <div class="list-item">
+          <ion-item lines="none">
+            <ion-thumbnail slot="start">
+              <DxpShopifyImg/>
+            </ion-thumbnail>
+            <ion-label class="ion-text-wrap">
+              primary identifier
+              <p>secondary identifier</p>
+            </ion-label>
+          </ion-item>
+          
+          <ion-label>
+            3
+            <p>{{ translate("systemic") }}</p>
+          </ion-label>
+
+          <ion-item lines="none">
+            <ion-label class="ion-text-center">
+              <ion-badge color="danger">{{ translate("not counted") }}</ion-badge>
+              <p>{{ translate("last counted 1 week ago") }}</p>
+            </ion-label>  
+          </ion-item>
+
+          <div class="tablet">
+            <ion-button fill="outline" color="success" size="small">
+              <ion-icon slot="icon-only" :icon="thumbsUpOutline" color="success"></ion-icon>
```

Comment:
No need to add color on icons, as it will by default inherit the color from the button